lib/rubocop/cop/performance/define_method.rb

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Performance
+      # This cop is used to identify usages of `module_eval` and suggest
+      # changing them to `define_method`.
+      class DefineMethod < Cop
+        MSG = 'Use `define_method` instead of `module_eval`.'
+
+        def on_send(node)
+          _, method, _ = *node
+
+          return unless method == :module_eval
```

Comment:
Won't this yield some false positives in its current form? After all you might use `module_eval` for something different than a method definition.